packages/react-icons/src/utils/wrapIcon.tsx

```diff
@@ -2,11 +2,15 @@ import * as React from "react";
 import { FluentIconsProps } from "./FluentIconsProps.types";
 import { useIconState } from "./useIconState";
 
-const wrapIcon = (Icon: (iconProps: FluentIconsProps) => JSX.Element, displayName?: string) => {
-    const WrappedIcon = (props: FluentIconsProps) => { 
+type IconWithRefProps = 
+    React.ForwardRefExoticComponent<FluentIconsProps & 
+    React.RefAttributes<SVGSVGElement>>;
+
+const wrapIcon = (Icon: IconWithRefProps, displayName?: string) => {
```

Comment:
please also test if this works for bundled icons - as far as I remember we render two icons next to each other - not sure which one should obtain the ref.